yarn-project/circuits.js/src/structs/public_circuit_public_inputs.ts

```diff
@@ -126,55 +126,55 @@ export class PublicCircuitPublicInputs {
   }
 
   /**
-   * Returns an empty instance.
-   * @returns An empty instance.
+   * Returns an default instance.
+   * @returns An default instance.
    */
-  public static empty() {
+  public static default() {
     return new PublicCircuitPublicInputs(
-      CallContext.empty(),
+      CallContext.default(),
       Fr.ZERO,
       makeTuple(RETURN_VALUES_LENGTH, Fr.zero),
-      makeTuple(MAX_NULLIFIER_READ_REQUESTS_PER_CALL, ReadRequest.empty),
-      makeTuple(MAX_NULLIFIER_NON_EXISTENT_READ_REQUESTS_PER_CALL, ReadRequest.empty),
-      makeTuple(MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_CALL, ContractStorageUpdateRequest.empty),
-      makeTuple(MAX_PUBLIC_DATA_READS_PER_CALL, ContractStorageRead.empty),
+      makeTuple(MAX_NULLIFIER_READ_REQUESTS_PER_CALL, ReadRequest.default),
+      makeTuple(MAX_NULLIFIER_NON_EXISTENT_READ_REQUESTS_PER_CALL, ReadRequest.default),
+      makeTuple(MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_CALL, ContractStorageUpdateRequest.default),
+      makeTuple(MAX_PUBLIC_DATA_READS_PER_CALL, ContractStorageRead.default),
       makeTuple(MAX_PUBLIC_CALL_STACK_LENGTH_PER_CALL, Fr.zero),
-      makeTuple(MAX_NEW_NOTE_HASHES_PER_CALL, SideEffect.empty),
-      makeTuple(MAX_NEW_NULLIFIERS_PER_CALL, SideEffectLinkedToNoteHash.empty),
-      makeTuple(MAX_NEW_L2_TO_L1_MSGS_PER_CALL, L2ToL1Message.empty),
+      makeTuple(MAX_NEW_NOTE_HASHES_PER_CALL, SideEffect.default),
+      makeTuple(MAX_NEW_NULLIFIERS_PER_CALL, SideEffectLinkedToNoteHash.default),
+      makeTuple(MAX_NEW_L2_TO_L1_MSGS_PER_CALL, L2ToL1Message.default),
       Fr.ZERO,
       Fr.ZERO,
       Fr.ZERO,
       Fr.ZERO,
-      Header.empty(),
-      AztecAddress.ZERO,
+      Header.default(),
+      AztecAddress.default(),
       RevertCode.OK,
     );
   }
 
-  isEmpty() {
-    const isSideEffectArrayEmpty = (arr: SideEffect[]) => isArrayEmpty(arr, item => item.isEmpty());
-    const isSideEffectLinkedArrayEmpty = (arr: SideEffectLinkedToNoteHash[]) =>
-      isArrayEmpty(arr, item => item.isEmpty());
-    const isFrArrayEmpty = (arr: Fr[]) => isArrayEmpty(arr, item => item.isZero());
+  isDefault() {
```

Comment:
Is there a reason that we cannot just do the same as the other cases where checking that the toBuffer generates the same as the default case?